src/api/resource.ts

```diff
@@ -0,0 +1,86 @@
+import Axios from 'axios';
+import { Dispatch } from 'redux';
+
+import {
+  createResourceFailure,
+  createResourceSuccess,
+  getResourceFailure,
+  getResourceSuccess,
+  updateResourceFailure,
+  updateResourceSuccess,
+} from '../redux/actions/resourceActions';
+import { Resource, ResourceFileMap, ResourceMetadata } from '../types/resource.types';
+import { API_BASE_URL, StatusCode } from '../utils/constants';
+
+export const createNewResource = (files: ResourceFileMap[], metadata: ResourceMetadata, owner: string) => {
+  const resource: Partial<Resource> = {
+    createdDate: new Date().toISOString(),
+    files,
+    metadata,
+    owner,
+  };
+
+  return async (dispatch: Dispatch) => {
+    Axios({
+      method: 'POST',
+      url: `/${API_BASE_URL.RESOURCES}`,
+      data: resource,
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'aplication/json',
+      },
```

Comment:
Antar vi vil ha `application/json` over alt. mulig å definere dette globalt?

Reply:
tenkte på det, men vet ikke om det ganger oss? blir jo litt mindre å skrive da

Reply:
Det kommer vel security i headerene etterhvert også, så vi får nok behov for å trekke dem ut